install.sh

```
@@ -0,0 +1,46 @@
+
+git fetch --tags
+git checkout 1.44.0
+
+
+zip -r mediawiki.zip mediawiki -9 -x "*/.git/*" "*/i18n/*" "*/tests/*"
+
+
+# user
+# password
+
+cd extensions
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)